demo/docker/docker-compose.yml

```diff
@@ -7,14 +7,17 @@ services:
     ports:
       - "3081:8080"
     environment:
-      NUSSKNACKER_CONFIG_FILE: ${NUSSKNACKER_CONFIG_FILE:-/opt/nussknacker/conf/application.conf}
+      #multiple, comma separated, config files can be used. They will be merged in order, via HOCON fallback mechanism
```

Comment:
probably should be part of some documentation

Reply:
Yes, it will be part of configuration documentation, but I think here it's also ok to have it?